crypto_sign/sphincs-haraka-128f-robust/clean/address.h

```diff
@@ -1,13 +1,13 @@
-#ifndef SPX_ADDRESS_H
-#define SPX_ADDRESS_H
+#ifndef PQCLEAN_SPHINCSHARAKA128FROBUST_CLEAN_ADDRESS_H
+#define PQCLEAN_SPHINCSHARAKA128FROBUST_CLEAN_ADDRESS_H
 
 #include <stdint.h>
 
-#define SPX_ADDR_TYPE_WOTS 0
-#define SPX_ADDR_TYPE_WOTSPK 1
-#define SPX_ADDR_TYPE_HASHTREE 2
-#define SPX_ADDR_TYPE_FORSTREE 3
-#define SPX_ADDR_TYPE_FORSPK 4
+#define PQCLEAN_SPHINCSHARAKA128FROBUST_CLEAN_ADDR_TYPE_WOTS 0
```

Comment:
Why namespace these constants?

Reply:
They always had an SPX_ prefix; which was also the prefix I chose to add to all symbols. They simply got namespaced with all the rest.

Reply:
ok, I don't think this is necessary or consistently done in the other implementations in PQClean, but it doesn't hurt.